connection.go

```diff
@@ -208,3 +208,34 @@ func escapeValue(v string) []byte {
 
 	return buf.Bytes()
 }
+
+func (c *conn) Appender(schema string, table string) (*appender, error) {
+	if c.closed {
+		panic("database/sql/driver: misuse of duckdb driver: Appender after Close")
+	}
+
+	var schemastr *(C.char)
+	if schema != "" {
+		schemastr = C.CString(schema)
+		defer C.free(unsafe.Pointer(schemastr))
+	}
+
+	tablestr := C.CString(table)
+	defer C.free(unsafe.Pointer(tablestr))
+
+	var a C.duckdb_appender
+	if state := C.duckdb_appender_create(*c.con, schemastr, tablestr, &a); state == C.DuckDBError {
+		return nil, fmt.Errorf("can't create appender")
+	}
+
+	return &appender{c: c, schema: schema, table: table, appender: &a}, nil
+}
+
+func Appender(dbconn driver.Conn, schema string, table string) (*appender, error) {
```

Comment:
I would suggest to add a `NewAppenderFromConn(conn driver.Conn, schema, table string) (*Appender, error)` method to the `appender.go` and move the code from `(c *conn) Appender()` there. This would keep the current DuckDB implementation close to the original `database/sql` design idea and all extensions, like the appender, would be more separated. What do you think about this?